data/db_demo.yaml

```diff
@@ -1154,6 +1154,21 @@ analysis_service:
     group_ids:
       - =program_A
       - =program_B
+  - name: "OpenAI Summarization & Embedding"
+    display_name: "OpenAI Summarization & Embedding"
+    description: "Use OpenAI tools to summarize sources"
+    version: "1.0"
+    contact_name: "Josh Bloom"
+    url: "http://localhost:7003/summarize"
+    authentication_type: "none"
+    analysis_type: "meta_analysis"
+    input_data_types: ["redshift", "annotations", "classifications", "comments"]
```

Comment:
Note: we currently dont use annotations in the summaries

Reply:
This will be good to evolve towards, given these are some of the interesting cross-match and potentially ML score information.

Reply:
done